docker/Dockerfile.cuda

```diff
@@ -42,14 +35,23 @@ ENV LANG=C.UTF-8 \
     UV_CONSTRAINT=/tmp/constraints.txt \
     VIRTUAL_ENV=/opt/vllm
 
-# Update base packages
-RUN dnf -q update -y && dnf clean all
-
 # Install base packages and EPEL in single layer
+COPY docker/scripts/cuda/common/package-utils.sh /tmp/package-utils.sh
+COPY docker/scripts/cuda/builder/builder-package-mappings.json /tmp/builder-package-mappings.json
 COPY docker/scripts/cuda/builder/install-base-packages.sh /tmp/install-base-packages.sh
-RUN chmod +x /tmp/install-base-packages.sh && \
-    /tmp/install-base-packages.sh && \
-    rm /tmp/install-base-packages.sh
+RUN if [ "${TARGETOS:-rhel}" = "ubuntu" ]; then apt-get update -qq && apt-get install -y jq; else dnf -q install -y jq; fi && \
```

Comment:
I think the installs for the `jq` should be moved to `install-base-packages.sh` as it is a base package. Also in the case of RHEL we still need to call `dnf update -y`?